version/version.go

```diff
@@ -0,0 +1,50 @@
+package version
+
+import (
+	_ "embed"
+	"fmt"
+	"strings"
+)
+
+var (
+	// The git commit that was compiled. These will be filled in by the
+	// compiler.
+	GitCommit string
+
+	// The next version number that will be released. This will be updated after every release
+	// Version must conform to the format expected by github.com/hashicorp/go-version
+	// for tests to work.
+	// A pre-release marker for the version can also be specified (e.g -dev). If this is omitted
+	// then it means that it is a final release. Otherwise, this is a pre-release
+	// such as "dev" (in development), "beta", "rc1", etc.
+	//go:embed VERSION
+	fullVersion string
+
+	Version, VersionPrerelease, _ = strings.Cut(fullVersion, "-")
+
+	// https://semver.org/#spec-item-10
+	VersionMetadata = ""
+
+	// The date/time of the build (actually the HEAD commit in git, to preserve stability)
+	// This isn't just informational, but is also used by the licensing system. Default is chosen to be flagantly wrong.
+	BuildDate string = "1970-01-01T00:00:01Z"
+)
+
+// GetHumanVersion composes the parts of the version in a way that's suitable
+// for displaying to humans.
+func GetHumanVersion() string {
```

Comment:
Is there a reason we cannot continue calling this `VersionString()`? The `Get*` part in particular is typically considered anti-pattern in Go method names and AFAICT we also use this version as part of the `terraform-ls version -json` output.

Reply:
I've reverted it to `VersionString`